deep_quoridor/src/quoridor_env.py

```diff
@@ -156,6 +156,7 @@ def _get_observation(self, agent_id):
         walls = self.walls.copy()
 
         return {
+            "my_turn": self.agent_selection == agent_id,
```

Comment:
If we add this to the observation (which I support) we should also add it to he observation space. Can you add that before this specific change is merged?